cfme/tests/services/test_pxe_service_catalogs.py

```diff
@@ -116,7 +117,8 @@ def catalog_item(appliance, provider, dialog, catalog, provisioning,
 
 
 @pytest.mark.rhv1
-@pytest.mark.meta(blockers=[GH('ManageIQ/integration_tests:7491')])
+@pytest.mark.meta(blockers=[GH('ManageIQ/integration_tests:7491'),
```

Comment:
Remove this BZ as its been verified, but keep the PR to remove the closed GH issues and remove the import. (already discussed in other comment threads I can't reply to)